### PR TITLE
update jenkins build scripts

### PR DESCRIPTION
**Pull request by akva2**
output is now an upstream and not a downstream

### CODE REVIEW

**Comment by akva2** [comment]
unless there are objections i will shamelessly self-merge this to stop the jenkins spam in 1h.

File: jenkins/build-pr.sh [hunk 1]
```diff
@@ -32,15 +34,15 @@ do
...
+echo "Building with ert=$ERT_REVISION opm-common=$OPM_COMMON_REVISION opm-parser=${upstreamRev[opm-parser]} opm-material=${upstreamRev[opm-material]} opm-output=${upstreamRev[opm-output]} opm-core=$sha1"
 
 build_opm_core
 test $? -eq 0 || exit 1
 
 # If no downstream builds we are done
 if ! grep -q "with downstreams" <<< $ghprbCommentBody
 then
-  cp serial/build-opm-material/testoutput.xml .
+  cp serial/build-opm-core/testoutput.xml .
```
Comment (atgeirr) [inline jenkins/build-pr.sh]: This is an unrelated bug fix, right?

Comment (akva2) [inline jenkins/build-pr.sh]: yes didn't think it worth the work to split

**Comment by atgeirr** [comment]
> to stop the jenkins spam

I have not got any spam, what kind have you been getting?

**Comment by akva2** [comment]
i get every failure on jenkins.

**Comment by atgeirr** [comment]
I thought I did -- apparently not.

**Comment by akva2** [comment]
probably not consistently. you can add yourself on the jenkins jobs, near the bottom.
